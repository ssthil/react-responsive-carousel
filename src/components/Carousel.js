import React from 'react';

const Carousel = ({ imgUrl, alt, userName }) => (
  <div className="image-holder">
    <div>
      <img src={imgUrl} alt={alt}  />
      <div className="user-name">{userName}</div>
    </div>
  </div>
)

export default Carousel;