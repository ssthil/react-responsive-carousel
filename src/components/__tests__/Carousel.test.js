import React from 'react';
import { shallow } from '../../enzyme';

import Carousel from "../Carousel";

describe("Carousel tests", () => {
  it("renders list of users image", () => {

    const userName = Array(20).fill(null);
    const wrapper = shallow(<Carousel userName={userName.user} />);

    console.log(wrapper.debug());
    // Expect the wrapper object to be defined
    expect(wrapper.debug()).toMatchSnapshot()
    expect(wrapper.find('.carousel-container')).toBeDefined();
    // expect(wrapper.find('.carousel-container')).toHaveLength(userName.length);
  })
})
