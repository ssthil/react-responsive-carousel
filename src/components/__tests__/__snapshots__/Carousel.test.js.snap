// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel tests renders list of users image 1`] = `
"<div className=\\"image-holder\\">
  <div>
    <img src={[undefined]} alt={[undefined]} />
    <div className=\\"user-name\\" />
  </div>
</div>"
`;
